packages/aml-checklist/src/services/documents.ts

```diff
@@ -1,27 +1,37 @@
-import { fetcherWithBlob } from '@reapit/elements'
+import { fetcherWithBlob, fetcher } from '@reapit/elements'
 import { URLS } from '@/constants/api'
 import { initAuthorizedRequestHeaders } from '@/utils/api'
 import { logger } from 'logger'
 
 export const downloadDocument = async (documentId: string) => {
   try {
     const headers = await initAuthorizedRequestHeaders()
-    headers['accept'] = 'application/octet-stream'
 
-    const response = await fetcherWithBlob({
-      url: `${URLS.documents}/${documentId}/download`,
-      api: window.reapit.config.platformApiUrl,
-      method: 'GET',
-      headers: headers,
-    })
+    const [document, documentBlob] = await Promise.all([
```

Comment:
I think you should change the name of the document, it'll be confuse with global object document.getElement ...